.gitignore

```diff
@@ -21,3 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+node_modules
```

Comment:
Was it not ignored before? Doesn't seem so.

Reply:
I see it is different from `/node_modules` above on line 4. But is it needed?

Reply:
For whatever reason, VSCode did not catch that and suggested me to add `node_modules` to `.gitignore`. But since it was already there, I agree that there's no reason to duplicate it